app/scripts/modules/netflix/fastProperties/wizard/createFastPropertyWizard.controller.ts

```diff
@@ -0,0 +1,81 @@
+import {module} from 'angular';
+
+import { FAST_PROPERTY_DETAILS_COMPONENT } from './propertyDetails/propertyDetails.componet';
+import { FAST_PROPERTY_SCOPE_COMPONENT } from './propertyScope/propertyScope.componet';
+import { FAST_PROPERTY_REVIEW_COMPONENT } from './propertyReview/propertyReview.componet';
+import { FAST_PROPERTY_STRATEGY_COMPONENT } from './propertyStrategy/propertyStrategy.componet';
+import { PIPELINE_CONFIG_SERVICE } from 'core/pipeline/config/services/pipelineConfig.service';
+import { PROPERTY_MONITOR_SERVICE } from './monitor/propertyMonitorService';
+import { PropertyCommandType }from '../domain/propertyCommandType.enum';
+import {PropertyPipeline} from '../domain/propertyPipeline.domain';
+import IModalServiceInstance = angular.ui.bootstrap.IModalServiceInstance;
+import {PropertyCommand} from '../domain/propertyCommand.model';
+
+class CreateFastPropertyWizardController {
+
+  public command: PropertyCommand = new PropertyCommand();
+  public loading = false;
+  public propertyMonitor: any;
+
+  static get $inject() { return [
+    '$scope',
+    '$uibModalInstance',
+    'title',
+    'applicationName',
+    'pipelineConfigService',
+    'propertyMonitorService',
+  ]; }
+
+  public state: { [stateKey: string]: boolean } = {
+    loading: false,
+  };
```

Comment:
looks like `state` is not used - I see `ctrl.loading` but no references to `ctrl.state` in the template